### DISCUSSION TITLE
THE WELCOME LINK HAS ALREADY BEEN USED.

### DISCUSSION THREAD

**Discussion by StaffCollab** [spatie/laravel-welcome-notification#48] [Q&A]
I implemented the spatie package "laravel-welcome-notification" and whenever anyone clicks on the link sent in the email I'm getting the 403 error

THE WELCOME LINK HAS ALREADY BEEN USED.

I did a custom notification, and I think that in the beginning, it did work, and then suddenly, it stopped working.

**Comment by malle-pietje** [comment]
Share your custom code, otherwise it’s a challenge for others to figure out what is wrong…
